catroid/src/main/java/org/catrobat/catroid/utils/TouchUtil.java

```diff
@@ -72,11 +72,20 @@ public static void touchUp(int pointer) {
 		currentlyTouchingPointersToTouchIndex.delete(pointer);
 	}
 
-	public static boolean isFingerTouching(int index) {
+	public static double isTouching() {
+		for (int i = 0; i < isTouching.size(); i++) {
+			if (isTouching.get(i)) {
+				return 1d;
+			}
+		}
+		return 0d;
+	}
+
+	public static double isFingerTouching(double index) {
```

Comment:
Please do not change the bahaviour of the function here. There are two stageIsTouched sensors, one with parameter for the index, one without parameter. You were correct with making a new function for the one without parameter. The one with parameter however should stay the same and give information if a specific finger is still on the screen.